Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
+This is how you would change the second line.
```

Comment:
A very very long thread. I should receive an event by this time, I hope? Will I? Shall I? Well, we will see.